regression/cbmc-java/swap2/test.desc

```diff
@@ -1,6 +1,6 @@
 CORE
 org/springframework/build/gradle/MergePlugin$1$_execute_closure1$_closure2.class
---function "org.springframework.build.gradle.MergePlugin\$1\$_execute_closure1\$_closure2.\$getCallSiteArray:()[Lorg/codehaus/groovy/runtime/callsite/CallSite;"
+--function 'org.springframework.build.gradle.MergePlugin$1$_execute_closure1$_closure2.$getCallSiteArray:()[Lorg/codehaus/groovy/runtime/callsite/CallSite;'
```

Comment:
I'm in relative awe at you tracking this down btw, glad we found a combination of escaped characters and quotes that is cross platform compatible...